### DISCUSSION TITLE
Is there any release jar for antlr4.10.1 that can be run over JDK8?

### DISCUSSION THREAD

**Discussion by hagen666** [antlr/antlr4#3893] [Q&A]
Is there any release jar for antlr4.10.1 that can be run over JDK8? 
The released jar on https://github.com/antlr/website-antlr4/tree/gh-pages/download can be run only over JDK11.

The way I found is changing the pom.xml and compiling the source code by myself.
Is there any website or url that I can download a 4.10.1 jar that can be run over JDK8?

**Comment by ericvergnaud** [comment]
are you referring to antlr runtime or antlr complete (with tool) ?

**Reply by hagen666** [reply]
antlr tools,  not the runtime.

**Comment by ericvergnaud** [comment]
Hi,No the tool now requires jre 11.EricEnvoyé de mon iPhoneLe 28 oct. 2022 à 09:13, hagen666 ***@***.***> a écrit :﻿
antlr tools,  not the runtime.

**Reply by hagen666** [reply]
I can  cross recompile it using JDK 11 and run it using JAVA 8. Maybe it just recompile.

**Comment by ericvergnaud** [comment]
The tool won’t run in java 8.
The standalone runtime still runs using java 8, but that’s a different jar.
